iceoryx_posh/test/moduletests/test_popo_chunk_queue.cpp

```diff
@@ -188,22 +186,19 @@ TYPED_TEST(ChunkQueue_test, AttachSecondConditionVariable)
     ConditionVariableData condVar1;
     ConditionVariableData condVar2;
     ConditionVariableWaiter condVarWaiter1{&condVar1};
-    ConditionVariableWaiter condVarWaiter2{&condVar1};
+    ConditionVariableWaiter condVarWaiter2{&condVar2};
 
-    auto ret1 = this->m_popper.setConditionVariable(&condVar1);
-    EXPECT_TRUE(ret1);
-
-    auto ret2 = this->m_popper.setConditionVariable(&condVar2);
-    EXPECT_FALSE(ret2);
+    this->m_popper.setConditionVariable(&condVar1);
+    this->m_popper.setConditionVariable(&condVar2);
```

Comment:
What happens here? I think the old conditionvariable is just overwritten (i.e. m_popper is now associated with condVar2).
This would explain the change in the expects below (true <-> false switch).

Reply:
Old condition variable is just overwritten.